e2e/pre.sh

```diff
@@ -12,7 +12,7 @@ else
     echo "CI_SHA1: $CI_SHA1"
 fi
 
-yq w -d5 -i deploy/all.yaml 'spec.template.spec.containers[0].image' "quay.io/reactiveops/rbac-manager:dev-$CI_SHA1"
-cat deploy/all.yaml
+yq w -d5 -i deploy/3_deployment.yaml 'spec.template.spec.containers[0].image' "quay.io/reactiveops/rbac-manager:dev-$CI_SHA1"
```

Comment:
```suggestion
yq w -i deploy/3_deployment.yaml 'spec.template.spec.containers[0].image' "quay.io/reactiveops/rbac-manager:dev-$CI_SHA1"
```

I think this should fix it.

Reply:
Ah! Thanks!